targets/csharp/make.js

```diff
@@ -25,18 +25,28 @@ exports.makeCombinedAPI = function (apis, sourceDir, apiOutputDir) {
         libname: "All",
         sdkDate: sdkGlobals.sdkVersion.split(".")[2],
         sdkVersion: sdkGlobals.sdkVersion,
-        sdkYear: sdkGlobals.sdkVersion.split(".")[2].substr(0, 2)
+        sdkYear: sdkGlobals.sdkVersion.split(".")[2].substr(0, 2),
+        azureSdk: false
     };
 
+    if (sdkGlobals.buildFlags.includes("azure")) {
+        locals.azureSdk = true,
+        locals.hasServerOptions = false,
+        locals.extraDefines = ";NETFX_CORE;SIMPLE_JSON_TYPEINFO;DISABLE_PLAYFABCLIENT_API"
```

Comment:
Nit:
;NETFX_CORE;SIMPLE_JSON_TYPEINFO; could be in some constant at the top, since it's reused now.